src/main/java/uk/gov/companieshouse/api/accounts/controller/Period.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+import java.util.Arrays;
+
+public enum Period {
```

Comment:
Can we rename this to `AccountingPeriod` to be more explicit? Keep the private `period` field named as is - Sonar doesn't like conflicting field and object names

Reply:
renamed Period  class to AccountingPeriod